Story-Chief/frontend-test-amanijuanagustin
# added filter by query functionality

## Description

Added a new Input so user is able to filter by query.
If the query is included in the description and in the title if the event it will be filtered



## Closes issue(s)



## Testing Instructions

Type in the Search input.
You have to see the event that matches with the query in the calendar






## Screenshots

https://user-images.githubusercontent.com/47245109/197993141-d71c721c-023f-425a-ac66-46525d935880.mov
